packages/babel-plugin-minify-dead-code-elimination/__tests__/dead-code-elimination-test.js

```diff
@@ -791,6 +791,30 @@ describe("dce-plugin", () => {
     expect(transform(source).trim()).toBe(expected);
   });
 
+  it("should evaluate conditional expressions 3", () => {
+    const source = "'foo' ? a() : b();";
+    const expected = "a();";
+    expect(transform(source).trim()).toBe(expected);
+  });
+
+  it("should evaluate conditional expressions 4", () => {
+    const source = "null ? a() : b();";
+    const expected = "b();";
+    expect(transform(source).trim()).toBe(expected);
+  });
+
+  it("should evaluate conditional expressions 5", () => {
+    const source = "'foo' === 'foo' ? a() : b();";
```

Comment:
I wasn't sure if the assumption was that `'foo' === 'foo'` would be reduced to `true` by constant folding, but I didn't see any tests for that in the constant folding plugin.

So if you prefer I can move these two to `constant-folding-test.js`